cohort-util/src/main/java/com/ibm/cohort/valueset/CodeSystemLookup.java

```diff
@@ -9,8 +9,14 @@
 package com.ibm.cohort.valueset;
 
 
+import com.ibm.cohort.annotations.Generated;
+
 //taken from cqf-tooling's CodeSystemLookupDictionary.java
+//tagged to avoid coverage checks
+@Generated
```

Comment:
If the class is tagged with `@Generated`, then do we need `cohort-util/src/test/java/com/ibm/cohort/valueset/CodeSystemLookupTest.java`?

Reply:
The test was added before the generated tag was added... it feels silly to test all the cases of the switch statement, but if you don't it significantly drags down the coverage.  I'm ok deleting the test, but it also doesn't feel bad to have around.

Reply:
The code isn't technically generated, but we don't have another way of ignoring classes for test coverage purposes.

Reply:
> The code isn't technically generated, but we don't have another way of ignoring classes for test coverage purposes.

Understood. As long as including both the `@Generated` annotation and the test class was done on purpose then that works for me. I just wanted to make sure there wasn't a cleanup step missed after the annotation was added.